lib/renderer/init.ts

```diff
@@ -203,7 +205,7 @@ for (const preloadScript of preloadScripts) {
     if (!isParentDir(getAppPath(), fs.realpathSync(preloadScript))) {
       throw new Error('Preload scripts outside of app path are not allowed')
     }
-    __non_webpack_require__(preloadScript)  // eslint-disable-line
+    Module._load(preloadScript)  // eslint-disable-line
```

Comment:
what lint are we disabling here?

Reply:
Can't remember, might be a leftover from the `__non*`